.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "meta-ptx"]
 	path = meta-ptx
 	url = https://github.com/pengutronix/meta-ptx.git
-	branch = langdale
+	branch = master
```

Comment:
Note that we are going to remove compatibility with `mickledore` in master soon, so we should probably change this to `mickledore`.